src/device-state.ts

```diff
@@ -578,10 +578,17 @@ export async function getStatus(): Promise<DeviceStatus> {
 	// at that point we can filter non-system apps leaving a single user app.
 	// After this, for true multi-app, we will need to report our status back in a
 	// different way, meaning this function will no longer be needed
+
+	// TODO: in tri-app, we need to filter first the user app from the host and supervisor apps
+	// on the to get the correct user app id
```

Comment:
```suggestion
	// on the state to get the correct user app id
```

Can also remove the comment block above it, they appear to be saying the same thing